vehicle/mkvi/software/sensing/sensing.h

```diff
@@ -0,0 +1,37 @@
+#include "libs/adc/api.h"
+#include "libs/gpio/api.h"
+#include "libs/gpio/pin_defs.h"
+#include "libs/timer/api.h"
+
+/* ATMega 16m1 Pins */
+gpio_t DEBUG_LED = PB0;
+gpio_t WHEEL_SPEED_LEFT = PB2;
+gpio_t WHEEL_SPEED_RIGHT = PD6;
+
+enum { counter_left = 0 };
+enum { counter_right = 0 };
```

Comment:
Are these supposed to be within one enum?

Reply:
Oh I kept reading, seems like these are just counters? Is there a reason why you didn't just have these as 8 or 16 bit integers?

Reply:
There was something with the interrupts that made @amit-kumarh suggest just making them global variables. It makes the syntax easier (?)

Reply:
They can be global variables and `uint8_t`s or something like that would be what I would say. I would also for readability sake maybe define this in the .c file, since it isn't really configuration? up to you

edit: also potentially see my other comment about getting rid of these